app/src/main/java/com/example/mycalculator/mvp/CalculatorModelImpl.java

```diff
@@ -0,0 +1,17 @@
+package com.example.mycalculator.mvp;
+
+public class CalculatorModelImpl implements CalculatorContract.CalculatorModelContract {
+
+    private String firstOperator;
```

Comment:
this should call firstOperand as will be the first number not the operator

Reply:
Done. Thx!